File: service/IhChegou.Repository.Model/Model/Stores/Store.cs
using IhChegou.Repository.Model.Clients;
using IhChegou.Repository.Model.Orders;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IhChegou.Repository.Model.Stores
{
     public class Store
    {
        public virtual int Id { get; set; }
        public virtual string Name { get; set; }
        public virtual string Email { get; set; }
        public virtual string PasswordHash { get; set; }
        public virtual string Document { get; set; }
        public virtual float Profit { get; set; }
        public virtual double MaxDistance { get; set; }
        public virtual int Address_Id { get; set; }

        //public virtual IList<PaymentMethod> AvailablePayments { get; set; }
        //public virtual IList<Delivery> AvaibleDeliveries { get; set; }

    }
}

File: app/veloz-farma-app/src/app/models/store.ts
import { Address } from './client';
import { PaymentMethod, Delivery } from './order';

export class Store {
    Name: string;
    Email: string;
    Password: string;
    Address: Address;
    AvailablePayments: Array<PaymentMethod>;
    AvaibleDeliveries: Array<Delivery>;
    Proximity: number;
    MaxDistance: number;
}